app/assets/javascripts/blob/upload_file.js.coffee

```diff
@@ -0,0 +1,35 @@
+Dropzone.options.blobFileDropzone = false;
+
+$(document).on "page:load", ->
+      $("#blobFileDropzone").dropzone()
+
+Dropzone.options.blobFileDropzone = {
+  autoProcessQueue: false,
+  clickable: true,
+  uploadMultiple: false,
+  paramName: "file",
+  parallelUploads: 1,
+  maxFiles: 1,
+  addRemoveLinks: true,
+  previewsContainer: '.dropzone-previews',
```

Comment:
I suggest adding a border around the file upload box. http://www.dropzonejs.com/ has an example.